doc/source/microsoft/step-zero-azure.rst

```diff
@@ -37,7 +43,7 @@ Step Zero: Kubernetes on Microsoft Azure Container Service (AKS)
      to log in.
 
 
-2. Activate the correct subscription. Azure uses the concept
+1. Activate the correct subscription. Azure uses the concept
```

Comment:
Could you change these to using `#.` instead of the numbers themselves? At least on my computer, these are all rendering as "1"s instead of going sequentially through the numbers. I think @yuvipanda has used a pattern like:

```
#. foo
#. bar
```

to get it to cycle through them properly in rST